libs/packages/components/src/lib/tree-table/tree-table.component.html

```diff
@@ -30,7 +30,8 @@
   let-parentRow="parentRow"
 >
   <tr
-    role="row"
+    role="treeitem"
+    aria-checked="false"
```

Comment:
Why aria-checked?

Reply:
TR (role=treeitem) does not have an aria-checked attribute is what showed up, so i set it to false to indicate it isn't checkable

Reply:
Showed up where? Setting aria-checked=false does not indicate that is cannot be clicked, it indicates that the TR is not currently clicked. Given the description of [aria-checked](https://w3c.github.io/aria/#aria-checked) from WAI-ARIA, I do not think noting anything about being checked or un-checked makes sense for the tree-table component.

Reply:
it shows up in AMP if I remove the aria-checked=false.
I just added it because it showed up in amp, I can remove it if it doesn't make sense for it to be there

<img width="629" alt="Screen Shot 2023-08-14 at 9 36 11 AM" src="https://github.com/GSA/sam-design-system/assets/118462746/fe68b3f4-0245-4c47-8780-5adeb31a7ab8">

Reply:
AMP reporting it usually is an indicator that something is wrong, but it would not be the first time it is wrong. Based on what aria-checked represents, it doesn't make sense to me having that aria-property on our tree-table. But if you think it should be set, for a reason other than the automated tool said it should be (hint: why does the automated tool think that?), I would be interested in hearing your rationale and can be swayed to be okay with that property being set.

Reply:
yeah, now looking at it again, it doesnt make sense to have aria-checked, I guess I just added it so that amp doesnt complain, but amp error messages are misleading sometimes, so I will just remove the aria-checked attribute.